src/Components/Home.jsx

```diff
@@ -21,28 +22,24 @@ const Home = () => {
           count += 1;
           return (
             <li id={data.volumn} key={count}>
+              <NavLink to={`/details/:${data.symbol}/:${count}`}>
+                <button type="button" className="Button input-submit" onClick={() => dispatch(IC(data.symbol))}><FaAngleRight /></button>
+              </NavLink>
```

Comment:
- [ ] One of the project requirements is that when the app loads, `the web app shows the list of categories that could be filtered out by some parameter, for example by the category name`. What it means is that, we should add a means of filtering (search), when users enter the name of one of the items, the item comes. I think you missed this special requirement.

![search](https://user-images.githubusercontent.com/53124548/217784975-5f873b4e-faaa-4934-b0e7-85d9643d15b4.png)

Reply:
Well resolved ✔️